app/models/user.rb

```diff
@@ -34,7 +36,7 @@ def omniauth_tokens(auth)
 
   def potential_swap_users(number = 5)
     # Clear out swaps every few hours to keep the list fresh for people checking back
-    potential_swaps.where(["created_at < ?", DateTime.now - 2.hours]).destroy_all
+    potential_swaps.where(["created_at < ?", DateTime.now - swap_validity_period]).destroy_all
```

Comment:
Ah no sorry, this should not be touched.  Expiry of confirmed swaps is what we want to parameterise, and that's a totally different thing to expiry of *potential* swaps.

Reply:
> a totally different thing to expiry of potential swaps.

I obviously haven't got my head round what the app does with all the data yet, apologies. Will fix obviously.

Reply:
Fix pushed